README.md

```diff
@@ -20,4 +20,17 @@ Hack hack hack. Save and your target will be rebuilt.
 
 Right now this repo supports `build`, `test`, and `run`.
 
+## Additional notes
+
+### What about the `--watchfs` flag?
+
+Bazel has a flag called `--watchfs` which, according to the bazel source does
+
+> If true, Bazel tries to use the operating system's file watch service
+> for local changes instead of scanning every file for a change
+
+Unfortunatel as of May 22, 2017, this option does not rebuild the project on
```

Comment:
After more though I would just scrap the "as of May...", it will always be like that IMO.